deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -428,12 +436,15 @@ def search(
             embedding_tensor (str): Name of tensor with embeddings. Defaults to "embedding".
             return_tensors (Optional[List[str]]): List of tensors to return data for. Defaults to None, which returns data for all tensors except the embedding tensor (in order to minimize payload). To return data for all tensors, specify return_tensors = "*".
             return_view (bool): Return a Deep Lake dataset view that satisfied the search parameters, instead of a dictionary with data. Defaults to False. If ``True`` return_tensors is set to "*" beucase data is lazy-loaded and there is no cost to including all tensors in the view.
+            deep_memory (bool): Whether to use the Deep Memory model for improving search results. Defaults to False if deep_memory is not specified in the Vector Store initialization.
+                If True, the distance metric is set to "deepmemory_norm", which represents the metric with which the model was trained. The search is performed using the Deep Memory model. If False, the distance metric is set to "COS" or whatever distance metric user specifies.
```

Comment:
why is it a `deepmemory_norm`, technically it is wrong to name it a `norm` better to say `deepmemory_distance` or just `deep_memory`?

Reply:
changed to `deepmemory_distance `